.travis.yml

```diff
@@ -2,15 +2,18 @@ language: python
 
 python:
   - 2.7
+  - 3.6
 
 env:
   - TOXENV=django18
```

Comment:
Do we still need to test Django 1.8 support?

Reply:
IMO, no. Removed.